CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ### [ [>](https://github.com/webdriverio-community/wdio-intercept-service/tree/v.next) ] v.next / <DATE>
 - Intercept HTTP requests upon initiation, rather than completion (thanks @tehhowch)
 
+### [ [>](https://github.com/webdriverio-community/wdio-intercept-service/tree/v4.1.11) ] 4.1.11 / 11/02/2022
+- Wrap xhr::abort and add tests for angular (thanks @lildesert)
```

Comment:
Ok, I think I didn't quite well understand on the first try. I also added `v4.2.1` since it was missing. https://github.com/webdriverio-community/wdio-intercept-service/pull/225/commits/e1704980741e5213b46d7647d7e3638f72268abf

Reply:
Thanks!

(This repo is still coming up to speed on its new release process; probably a new workflow to request tests for code changes and changelog updates for anything else would go a long way.

I think `release-it` can be customized to work with the format used, but that's for another PR.)